config/prometheus/monitor.yaml

```diff
@@ -5,12 +5,10 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-    app.kubernets.io/name: servicemonitor
-    app.kubernetes.io/instance: controller-manager-metrics-monitor
+    app.kubernets.io/name: controller-manager-metrics-monitor
```

Comment:
It would be better to use the labels from `default/kustomization.yaml` for all resources and remove all labels that are not specific for this resource. This makes it easier to avoid incorrect data in the future, e.g. when `app.kubernetes.io/component` is updated in `default/kustomization.yaml`. 
This applies to all resource-YAMLs.

Reply:
There was a typo in label key. Removed.